### PR TITLE
Fix list pages

### PR DESCRIPTION
**Pull request by kravets-levko**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix

## Description

Fixes getredash/redash#3535
Fixes getredash/redash#3539

### CODE REVIEW

**Comment by ranbena** [comment]
Another approach is to revoke the ability to click anywhere on the row and just have the row title clickable - which it already is! Wdyt?

(I don't think this feature is important enough to have our code unnecessarily complex and risky)

File: client/app/services/navigateTo.js [hunk 1]
```diff
@@ -10,3 +10,13 @@ export default function navigateTo(url, replace = false) {
     $rootScope.$applyAsync();
   }
 }
+
+export function handleNavigationEvent(event, url, replace = false) {
+  event.preventDefault();
+  event.stopPropagation();
+  if (event.ctrlKey || event.metaKey) {
```
Comment (gabrieldutra) [inline client/app/services/navigateTo.js]: I've never liked this sort of workaround to adapt `onClick` events to work as `href`. I.e.: I mostly use the middle mouse button for this purpose. However, I also don't have a better option for such cases :man_shrugging:

Comment (kravets-levko) [reply client/app/services/navigateTo.js]: Probably better solution would be to turn table rows into links somehow, but I have no idea how to do that...

Comment (kravets-levko) [reply client/app/services/navigateTo.js]: _(well, I know how, but it requires significant changes to markup)_

**Comment by kravets-levko** [comment]
@gabrieldutra @ranbena I changed this to use browser's native mechanism of handling `<a>` tags - at least in all my browsers it works perfectly (100% the same as using regular links). I'll test it in IE, can somebody test it on Mac?

**Comment by kravets-levko** [comment]
Okay, cannot test in IE because "Promise is undefined" :thinking:

**Comment by gabrieldutra** [comment]
> Okay, cannot test in IE because "Promise is undefined" thinking

Edge? (I believe it's even more important than IE :slightly_smiling_face:)

**Comment by arikfr** [comment]
I'm with @ranbena on this one: it's not worth the complexity. Let's have the different elements clickable (the name, groups, etc) using <a> tags and the disable full row clickability.

**Comment by kravets-levko** [comment]
@arikfr can you clarify what you did mean saying "Let's have the different elements clickable"? For example, should be something else clickable (in addition to name) on dashboard list page?

**Comment by arikfr** [comment]
@kravets-levko no, only name (and favorite control). On users page, the groups should be clickable.

**Comment by gabrieldutra** [comment]
> it's not worth the complexity

Perhaps a simpler helper method to track new tab events could be worthy, because iirc there is another part in Redash that does something similar (Fork query in new tab).

**Comment by kravets-levko** [comment]
@gabrieldutra In a case with "Fork query" we always need to open a new tab; but here we should mimic default browser's behavior which differs in different browsers/OS

**Comment by kravets-levko** [comment]
@arikfr It's very easy to remove this behavior; but could you try current solution first (in a case you'd like it :sweat_smile:)?

**Comment by kravets-levko** [comment]
@arikfr @ranbena @gabrieldutra Removed onClick behavior for table rows - my friend tried it in Edge and FF on Window - my fix with native events doesn't work there :disappointed:

**Comment by ranbena** [comment]
@kravets-levko perhaps also remove the `event.preventDefault(); event.stopPropagation();` in actions, like in UsersList?

**Comment by kravets-levko** [comment]
@ranbena Thanks a lot! Done

**Pull request merged**